de.dlr.sc.virsat.model.extension.requirements/src/de/dlr/sc/virsat/model/extension/requirements/csv/CsvFileReader.java

```diff
@@ -52,21 +57,57 @@ public CsvFileReader() {
 	 * @param dataEndLine the number of the last data line (starting from 0)
 	 */
 	public CsvFileReader(String separator, int headerLine, int dataStartLine, int dataEndLine) {
-		this.separator = separator + REGEX_ESCAPE_SPLIT;
+		this.separator = separator;
 		this.headerLine = headerLine;
 		this.dataStartLine = dataStartLine;
 		this.dataEndLine = dataEndLine;
 	}
 	
+	/**
+	 * Parse the given CSV file
+	 * @param filePath the path of the file as string
+	 * @throws IOException throws an exception if the file could not be loaded
+	 */
+	public boolean parseFile(String filePath) {
+		Path csvFilePath = Paths.get(filePath);
+		try {
+			fr = new FileReader(csvFilePath.toFile());
+			records = CSVFormat.EXCEL.withDelimiter(getSeparator().charAt(0)).parse(fr);

```

Comment:
It kind of feels misleading that we allow seperator to be any string but only use the first character to actually do the separation.